samples: cellular: http_update: modem DFU samples refactoring (#11036)
nrfconnect/sdk-nrf: samples/cellular/http_update/modem_full_update/README.rst

********

An |external_flash_size| of free space is required to perform a full modem update.

> Hence, only versions 0.14.0 and later of the nrf91 Series DK support this sample as the earlier versions do not have any external flash memory.

Reviewer: ```suggestion
Hence, for the nRF9160 DK, version 0.14.0 or higher is supported by the sample as the earlier versions do not have any external flash memory.
```

|test_sample|

1. Start the application and wait for a prompt for pressing a button.

> #. Press the **Button 1** button or type "download" in the terminal emulator to start downloading the update.
> #. Once the download has completed, the modem update procedure begins automatically.

Reviewer: ```suggestion
#. Press the **Button 1** button or type "download" in the terminal emulator to start downloading the update.
    Once the download is complete, the modem update procedure begins automatically.
```

> #. Once the modem update procedure has completed, the modem will be reinitialized to run the new firmware.

Reviewer: ```suggestion
   Once the modem update procedure is complete, the modem is reinitialized to run the new firmware.
```
